blog/views.py

```diff
@@ -21,35 +29,43 @@ def post_new(request):
             post = form.save(commit=False)
             post.author = request.user
             post.save()
-            return redirect('post_list')
+            post_data = serializers.serialize('json', [post, ])
+            return JsonResponse({'post_data': post_data}, status=HTTPStatus.CREATED)
+        else:
+            msg = {"message": "잘못된 입력입니다."}
+            return JsonResponse(msg, status=HTTPStatus.BAD_REQUEST)
```

Comment:
```suggestion
            return JsonResponse({"message": "잘못된 입력입니다."}, status=HTTPStatus.BAD_REQUEST)
```
msg는 불필요한 변수로 보이고, 바로 inline 해도 가독성에는 이상 없을 것 같아요~

Reply:
534e7ec - 반영했습니다!